### PR TITLE
Tab View for Essentially | Testing View

### PR DESCRIPTION
**Pull request by armintalaie**
a tab view focused on improving summarization development
- can check the status of servers
- multiple text sections and edit them
- rerun summary
- generates summary API time (for development and optimization purposes)
- word count for input and summary (for development and optimization purposes)
- feedback section for future
- request more keywords

some screenshots

Screenshot 2022-12-23 at 12 00 35 AM

Screenshot 2022-12-23 at 12 00 50 AM
Screenshot 2022-12-23 at 12 02 23 AM

Screenshot 2022-12-23 at 12 02 56 AM

- right click on any part of a page and a new web app option is added 🎉 

Screenshot 2023-01-02 at 9 27 21 PM

### CODE REVIEW

**Comment by armintalaie** [comment]
I still gotta do some code cleanup and styling updates
improvements for the future:
- retrieve more messages from servers
- connect feedback to a form API
- use env variables instead of absolute paths and URL
- host testing view on the web + APIs (AWS? Firebase?)
- add link input

File: backend/src/app.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import app from "./server";
+import serverless from "serverless-http";
```
Comment (Michaeltaoma) [inline backend/src/app.ts:2]: `serverless-http` is not listed in `package.json` and it looks like `serverless` is not being used

**Review by Michaeltaoma — COMMENTED** [review]
Should we exclude .DS_Store from version control? Also the testing page looks great!

File: backend/src/app.ts (hunk 1 above)
Comment (armintalaie) [reply backend/src/app.ts:2]: yep forgot to remove it. was testing some deployment stuff. thanks. will update

**Comment by armintalaie** [comment]
> Should we exclude .DS_Store from version control? Also the testing page looks great!

yeah i think we should go over some ground rules for ourselves. like preferences with what to include, some build config, etc

**Review by Michaeltaoma — APPROVED** [review]
LGTM!

**Pull request merged**